### ISSUE TITLE
Rewrite Compare page with newer version of React.js

### ISSUE THREAD

**Issue by natebass** [code4sac/openbudgetsac.org#60]
The Compare page uses an old version of React.js, which uses deprecated coding standards. Upgrading to the newer version of React.js would require a complete rewrite.

Work has already been started by @PlacidSerene https://github.com/PlacidSerene/openbudget-rework

**Comment by MonstersInc-sudo** [comment]
For this refactor are we looking to also update dependencies like ChartJS?

**Comment by natebass** [comment]
Yeah, we could update as many dependencies as we can to the current version. We would just need to check if there are any breaking API changes that would require us to update our code. I know that this is the case with React; it uses react-cli and makes some breaking changes to the way it declares its components.
